### ISSUE TITLE
ES5 examples

### ISSUE THREAD

**Issue by gabssnake** [react-dnd/react-dnd#26]
Greetings.  I have a use case in ES5 (not surprisingly), so it is great to have the mixing in ES5 as provided. However, I find it hard to parse ES6 in my head at the same time that I try to understand how the mixing works in the examples...

Is there any change you’ll eventually add ES5 examples :)

**Comment by gabssnake** [comment]
(Really late) 

Thanks for taking the time to answer with examples and some quality explanation, it actually helped me make the move to ES6.

With the project at hand, I wasn’t ready to enable ES6 (people weren’t open to changing settled parts of the software at that point in time), so I couldn’t test the examples with the workflow we were using.

Anyway, things went well.

I’ll be happy to make a PR with the examples in ES5 if there is interest in that.

**Comment by gaearon** [comment]
I'm currently doing a big rewrite and it's all ES6, so there's no point in going back to ES5. You can always look at the compiled code to see how to write the same in ES5 though.

ES6 is not destructive; you can enable the transformation and nobody will notice. Keeping using ES5 at this point is just unnecessary stagnation.
